### ISSUE TITLE
Problem with importing animation (SkyrimLE)

### ISSUE THREAD

**Issue by TokyoSU** [BadDogSkyrim/PyNifly#172]
Don't know if it's me doing something, following your tutorial for animation import/export,
I taken the dragon for example, i get these:
![not_centered](https://github.com/BadDogSkyrim/PyNifly/assets/77746747/afbe70eb-7298-46e0-acf8-d39115750e6f)
the animation is not at the root position and seem the skinning or bones is not correctly placed.
![bone_weird](https://github.com/BadDogSkyrim/PyNifly/assets/77746747/420140be-52e5-4a78-9a5c-fce2aff25597)
The jaw is missplaced and just pass through the neck :')

**Comment by BadDogSkyrim** [comment]
Dragons are a pain. Will check. I assume the dragon nif imported correctly? Which animation did you try to use?

**Comment by TokyoSU** [comment]
yes it was imported correctly, i used ground_bite.hkx but it happen with all animation, through it's not shifted when using the skeleton.hkx instead of skeleton.nif but it miss some bone and the animation got a bit weird :'(

**Comment by BadDogSkyrim** [comment]
I'll look. There's something wrong with auxbones animations too, so maybe this is the same bug.

**Comment by TokyoSU** [comment]
It happen with the dogs actor too.
actors -> canine -> dog specific files.

**Comment by TokyoSU** [comment]
Hope this bug with bones is solved, it's an awesome tools :D
